packages/core/src/node/runtimeModule/siteData/normalizeThemeConfig.ts

```diff
@@ -73,8 +73,17 @@ export function normalizeThemeConfig(
     const normalizeSidebarItem = (
       item: SidebarGroup | SidebarItem | SidebarDivider | SidebarSectionHeader | string,
     ): NormalizedSidebarGroup | SidebarItem | SidebarDivider | SidebarSectionHeader => {
-      // Meet the divider or section header, return directly
-      if (typeof item === 'object' && ('dividerType' in item || 'sectionHeaderText' in item)) {
+      // Meet the divider, return directly
```

Comment:
can we add an E2E test case for i18n.json?

Reply:
done